packages/synthchron-simulator/src/types/simulationTypes.ts

```diff
@@ -1,10 +1,43 @@
 // Configuration Types for the simulation
 export type Configuration = {
-  // TODO: Add configuration options here
+  configurationName?: string
   endOnAcceptingStateProbability: number
   minEvents?: number
   maxEvents?: number
   randomSeed: string
+  uniqueTraces?: boolean
+  terminationType:
+    | StandardConfigurationTerminationType
+    | CoverageTerminationType
+    | MaximumTracesTerminationType
+    | SpecifiedAmountOfTracesTerminationType
+}
+
+export enum terminationType {
+  Standard = 'standard',
+  MaximumTraces = 'maximumTraces',
+  Coverage = 'coverage',
+  SpecifiedAmountOfTraces = 'specifiedAmountOfTraces',
+}
+
+export type StandardConfigurationTerminationType = {
+  type: terminationType.Standard
+}
+
+export type CoverageTerminationType = {
+  type: terminationType.Coverage
+  coverage: number
+}
+
+export type MaximumTracesTerminationType = {
+  type: terminationType.MaximumTraces
+  maximumTraces: number
```

Comment:
The maximum traces parameter should not be a termination type, but rather a parameter. A user could always want to add a limit to the amount of traces generated, independent of termination type. This should therefore be part of the configuration.

Reply:
done